File: Rectangle.js
const Calculator = require("./Calculator");

module.exports = class Rectangle {
	constructor(length, width) {
		this.length = length;
		this.width = width;
	}

	area() {
		const calculator = new Calculator(this.length, this.width);
		return calculator.multiply();
	}

	perimeter() {
		const calculator = new Calculator(this.length, this.width);
		return 2 * calculator.plus();
	}
};

File: rectangle.cs
using Calculator;
namespace Rectangle
{
    class Rectangle
    {
        public int length, width;
        void Rectangle(int length, int width)
        {
            this.length = length;
            this.width = width;
        }
        public int Area(int a, int b)
        {
            return Calculator.Multiply(a, b);
        }
        public int Perimeter(int a, int b)
        {
            return Calculator.Sum(a, b);
        }
    }
}
